packages/picasso/src/Stepper/story/index.jsx

```diff
@@ -23,3 +23,4 @@ page
   .addExample('Stepper/story/Default.example.tsx', 'Default')
   .addExample('Stepper/story/WithoutLabels.example.tsx', 'Without Labels')
   .addExample('Stepper/story/Vertical.example.tsx', 'Vertical')
+  .addExample('Stepper/story/Overflow.example.tsx', 'Overflow')
```

Comment:
I think we can disable screenshotting for this story and put it to cypress side since the component needs hovering for taking happo screenshot.

```suggestion
  .addExample('Stepper/story/Overflow.example.tsx', {
      title: 'Overflow',
      takeScreenshot: false,
  })
```

Reply:
Done